### DISCUSSION TITLE
Reduce number of field dumps per unit time

### DISCUSSION THREAD

**Discussion by SengerM** [thliebig/openEMS-Project#249] [Q&A Usage]
Is it possible to reduce the number of frames dumped? Say, dump once every 1000 steps?

**Comment by Juleinn** [comment]
I don't think so, the dump interval is computed based on the Nyquist interval such that you can see the signal propagation in the smallest cells of your sim. You can increase the sampling with the oversampling parameter, but to decrease it you'd have to change the source code I think, but then you would start to see time-domain aliasing in your visualization. Your best bet is probably to increase the smallest cell size, if possible, to increase the timestep value.
